closure/goog/ui/emoji/emojipicker_test.js

```diff
@@ -810,6 +810,31 @@ testSuite({
     picker.dispose();
   },
 
+  testGetSelectedEmoji_urlPrefix() {
+    const defaultImg = 'http://example.com/../../demos/emoji/none.gif';
+    const picker = new EmojiPicker(defaultImg);
+    picker.setDelayedLoad(false);
+    picker.addEmojiGroup(emojiGroup1[0], emojiGroup1[1]);
+    picker.setUrlPrefix('http://example.com/')
```

Comment:
The CI seems to fail because you are missing a ; here, and the CI formatter wants to move the previous line onto this line.

Reply:
We'll fix this internally.

Reply:
OK!!